src/main/java/frc/robot/RobotContainer.java

```diff
@@ -170,7 +174,7 @@ private void configureButtonBindings() {
     
     controlBindings.babyBird().ifPresent(trigger -> trigger.whileTrue(
```

Comment:
Let's use `autoCommands.babyBird()` here. The reason for moving it into `autoCommands` was so we can reuse it.